README.MD

```diff
@@ -255,7 +255,24 @@ APIs we're issuing a special domain event:
 {
   "type": "DELETE_ALL",
   "tenant": "<the tenant name>"
-  "old": null,
-  "new": null
+}
+```
+
+## Reindex of instances
+
+Some consumers need to pull all instances from an existing database. There is
```

Comment:
Which consumers need to do this? Re-index seems like a term specific to the search context. Is it being used here to mean something more general?

Under what circumstances does this need to happen, given that the Kafka topics already contain a complete history of changes?

Reply:
> Under what circumstances does this need to happen, given that the Kafka topics already contain a complete history of changes?

When DB already contains some instances and we want to add them to mod-search, or when a breaking change is applied to elasticsearch index structure

Reply:
> When DB already contains some instances and we want to add them to mod-search

Is this going to be needed after mod-search is first enabled for it to have it's initial state?

> or when a breaking change is applied to elasticsearch index structure

In this case, why couldn't mod-search consume the original messages again?